Dockerfile

```diff
@@ -133,6 +133,7 @@ RUN ./install_bioc.r \
     spatial \
     treemap \
     umap  \
+    uwot \
```

Comment:
Typically changes to the dockerfile should be in a separate PR - can you make a separate PR for this change?